### ISSUE TITLE
Many provider attributes are not supported

### ISSUE THREAD

**Issue by jvanderhoof** [cyberark/terraform-provider-conjur#29]
## Steps to reproduce the issue
Assumptions 
- Conjur master is setup
- Valid credentials are available to the Terraform script as variables
1. Given the following TF template:
    ```
    # main.tf
    provider "conjur" {
      account = "${var.conjur_account}"
      appliance_url = "${var.conjur_appliance_url}"
      api_key = "${var.conjur_api_key}"
      login = "${var.conjur_login}"
      ssl_cert = "${var.conjur_ssl_cert}"
      ssl_cert_path = "${var.conjur_ssl_path}"
    }
    ```
1. Run `terraform apply`

## What's the expected result?
- The values for `account`, `appliance_url`, `ssl_cert`, and `ssl_cert_path` are used to connect to Conjur

## What's the actual result?
- Only the values for `api_key` and `login` are used to connect.  The rest of the values must be set using environment variables.

## Dev Notes
In tracking this bug down, I noticed the environment variables mentioned in the readme (ex. `export CONJUR_APPLIANCE_URL="https://localhost:8443"`) are the environment variables the Conjur GoLang library uses.  It looks like the convention for setting configuration with environment variables is more like:
```
Schema: map[string]*schema.Schema{
  "appliance_url": &schema.Schema{
    Type: schema.TypeString,
    Required: true,
    DefaultFunc: schema.EnvDefaultFunc("CONJUR_APPLIANCE_URL", nil),
    Description: "URL of the Conjur Appliance",
  },
  "account": &schema.Schema{
    Type: schema.TypeString,
    Required: true,
    DefaultFunc: schema.EnvDefaultFunc("CONJUR_ACCOUNT", nil),
    Description: "Conjur Account",
  },
  ...
}
```
which creates a more direct mapping between the environment variables and the provider attributes.  Does it make sense to update this technical debt as part of this effort?

**Comment by sgnn7** [comment]
Verified this bug - only `api_key` and `login` are supported. At the very least we should update the documentation about it.

**Comment by sgnn7** [comment]
PR: https://github.com/cyberark/terraform-provider-conjur/pull/59

**Comment by sgnn7** [comment]
Released as part of v0.4.0. Closing.
